docs/src/pages/en/themes/handling-variables.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Handling variables
+description: Learn how to handle variables in Media Chrome Themes
+layout: ../../../layouts/MainLayout.astro
+---
+
+Custom variables that the theme author defines can be provided by users of your
+theme through attributes and then used inside your theme with double curly brackets,
+for example `{{ username }}`
+
+If you would like to provide a fallback for a variable that might be empty,  
```

Comment:
Might be worth having the example below demonstrating this as well. Maybe by rendering a `videotitle` attribute that isn't provided on the `<media-theme>`?

Reply:
added this to the example